cmd/noms/splore/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["stage-3", "react"]
```

Comment:
I guess this works and you don't want to spend time on it?

The only transformers we really need at this point are Flow+JSX. Everything else is supported natively in Chrome.